### PR TITLE
Updated directory size and dates info table code

### PR DESCRIPTION
**Pull request by justinkadi**
Now saves output of command line to txt file, then creates data frame programmatically instead of copy/pasting output into tribble

### CODE REVIEW

File: report/arcticreport.Rmd [hunk 1]
```diff
@@ -414,7 +379,7 @@ tot <- tickets %>%
   select(id, Created, Resolved) %>% 
   gather(status, datetime, -id, na.rm = TRUE)
 
-names(tot) <- c("id", "status", "datetime")
+names(tot) <- c("id", "status", "datetime)"
```
Comment (jeanetteclark) [inline report/arcticreport.Rmd]: looks like a typo crept in here

File: report/arcticreport.Rmd [hunk 2]
```diff
@@ -188,8 +153,8 @@ Ask Robyn or the lead MetacatUI dev to give you access to Google Analytics if yo
 # Plots
 
 ```{r}
-qs <- "2023-05-01"
-qe <- "2023-07-31"
+qs <- "2023-08-01"
+qe <- "2023-10-31"
```
Comment (jeanetteclark) [inline report/arcticreport.Rmd]: can we change this so it isn't hard coded?

File: report/arcticreport.Rmd [hunk 3]
```diff
@@ -44,103 +44,68 @@ objs <- query_objects(cache_tolerance = 1000000)
...
 
 
+```{r}
+# Left joins the size_info and date_info tables
 datasets_add <- left_join(size_info, date_info) %>% 
-    mutate(size = size_kb*1024) %>% 
-    mutate(dateUploaded = as.Date(date)) %>% 
-    mutate(formatType = "DATA") %>% 
-    select(-size_kb,-date)
+  mutate(size = as.numeric(size_b)) %>%
```
Comment (jeanetteclark) [inline report/arcticreport.Rmd:99]: are we sure the units line up correctly here with what is in the table we eventually join to? based on the variable names it looks like they changed

**Review by jeanetteclark — CHANGES_REQUESTED** [review]

File: report/arcticreport.Rmd (hunk 3 above)
Comment (justinkadi) [reply report/arcticreport.Rmd:99]: The table we eventually join to is in bytes. In the original slower command for getting sizes, it returned the sizes in kb, then multiplied it by 1024 to convert the kb to bytes. In the new command Nick provided, the sizes are given in bytes, so they don't need to be multiplied by 1024 anymore.

File: report/arcticreport.Rmd (hunk 1 above)
Comment (justinkadi) [reply report/arcticreport.Rmd]: I changed this typo

File: report/arcticreport.Rmd (hunk 2 above)
Comment (justinkadi) [reply report/arcticreport.Rmd]: I pushed a new commit, where I added one line to change the current quarter to remove the hard coded dates

**Pull request merged**